src/mbgl/map/source.cpp

```diff
@@ -603,4 +606,58 @@ void Source::dumpDebugLogs() const {
     }
 }
 
+FeatureResults Source::featuresAt(const PrecisionPoint point, const uint8_t radius, const TransformState& transform) const {
```

Comment:
> Add more `const` modifiers in this function to clarify intent.

@kkaefer Can you clarify what you mean? 

> Why are we using a `uint8_t` for radius?

Fixed in https://github.com/mapbox/mapbox-gl-native/commit/d123545de9239bcab735c9ed77abbf785a1a6e9c. 

> `radius` appears to be measured in pixels. Does it account for pixel density?

It does, as the point and radius refer to screen points.